docs/src/content/common-props/common-props.md

```diff
@@ -226,6 +226,29 @@ class App extends React.Component {
 ReactDOM.render(<App/>, mountNode);
 ```
 
+## disableInlineStyles
+
+`type: boolean`
+
+The `disableInlineStyles` prop allows Victory components to work better with CSS classes or styled-components. By default, Victory provides inline styles to chart components, which will override any conflicting CSS styles. This flag will remove the inline styles, making it easier to provide custom styling for components via CSS.
```

Comment:
In this documentation, we should draw a distinction in the difference in behavior when providing the `disableInlineStyles` prop to a component like `VictoryBar` (i.e. inline styles are removed for both data and label components) vs supplying the prop to a component like `Bar`

Reply:
Done!